ansible/roles-infra/infra-common-ssh-config-generate/tasks/main.yml

```diff
@@ -38,18 +38,18 @@
     dest: "{{ ansible_ssh_config }}"
     marker: "##### {mark} ADDED BASTION PROXY HOST {{ bastion_hostname }} {{ env_type }}-{{ guid }} ######"
     content: |
-        Host {{ bastion_hostname }} {{ hostvars[bastion_hostname].shortname |d('')}}
-          Hostname {{ hostvars[bastion_hostname].public_dns_name }}
-          IdentityFile {{ ssh_key | default(infra_ssh_key) | default(ansible_ssh_private_key_file) | default(default_key_name)}}
-          IdentitiesOnly yes
-          User {{ remote_user }}
-          ControlMaster auto
-          ControlPath /tmp/{{ guid }}-%r-%h-%p
-          ControlPersist 5m
-          StrictHostKeyChecking no
-          ConnectTimeout 60
-          ConnectionAttempts 10
-          UserKnownHostsFile {{ansible_known_host}}
+         Host {{ bastion_hostname }} {{ hostvars[bastion_hostname].shortname |d('')}}
+         Hostname {{ hostvars[bastion_hostname].public_dns_name }}
+         IdentityFile {{ssh_key|default(infra_ssh_key)|default(ansible_ssh_private_key_file)|default(default_key_name)}}
+         IdentitiesOnly true
+         User {{ remote_user }}
+         ControlMaster auto
+         ControlPath /tmp/{{ guid }}-%r-%h-%p
+         ControlPersist 5m
+         StrictHostKeyChecking false
+         ConnectTimeout 60
+         ConnectionAttempts 10
+         UserKnownHostsFile {{ansible_known_host}}
```

Comment:
The content we want in the SSH config should read like:
```
Host ...
  Hostname ...
  IdentityFile...
...
```
It does look like there is a YAML lint indent issue, I think it should read:
```
     content: |
￼      Host {{ bastion_hostname }} {{ hostvars[bastion_hostname].shortname |d('')}}
￼        Hostname {{ hostvars[bastion_hostname].public_dns_name }}
```

Reply:
Fixed